rgbString/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "react-colorful-rgb-string",
```

Comment:
`build-packages` originally would've presented this as `"name": "react-colorful-rgbString"` but this does not follow [NPM's naming guide](https://docs.npmjs.com/files/package.json#name) as capital letters are not allowed. So I changed this into a hyphenated word.

Reply:
👍